src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -556,10 +556,12 @@ export default class FilingHistoryList extends Mixins(
           this.isEffectiveDatePast(effectiveDate)
         )
 
-        item.courtOrderNumber = filing.data.order?.fileNumber || ''
-        item.isArrangement = this.isEffectOfOrderPlanOfArrangement(filing.data.order?.effectOfOrder)
-        item.toLegalType = filing.data.alteration?.toLegalType || null
-        item.fromLegalType = filing.data.alteration?.fromLegalType || null
+        if (item.isCompletedAlteration) {
```

Comment:
How about a comment saying that this data is only valid for a completed alteration?

Reply:
Sure